src/createServer.js

```diff
@@ -1,3 +1,39 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const paramsCheck = require('./paramsCheck');
+const sendError = require('./sendError');
+
+const createServer = () => {
+  const server = http.createServer((request, response) => {
+    response.setHeader('Content-Type', 'application/json');
+
+    const { pathname, searchParams } = new URL(request.url, `http://${request.headers.host}`);
+    const text = pathname.slice(1);
+    const toCase = searchParams.get('toCase');
```

Comment:
Would be better to create a separate function for it
```suggestion
    const { text, toCase } = ...(....)
```